docs/pages/reference/cli.mdx

```diff
@@ -4,50 +4,56 @@ description: Detailed guide and reference documentation for Teleport's command l
-- [tctl](#tctl): An administrative tool that can configure the Teleport Auth Service.
+- [teleport](#teleport): A process and CLI tool that supports the Teleport access plane.
+- [tsh](#tsh): A CLI tool that allows end users to interact with Teleport sessions and list cluster information.
+- [tctl](#tctl): An administrative CLI tool that can configure the Teleport Auth Service.
+- [tbot](#tbot): A CLI tool that supports Machine ID on Teleport, used to apply short lived certificates to service accounts (e.g, a CI/CD server).
 
 (!docs/pages/includes/permission-warning.mdx!)
 
 (!docs/pages/includes/backup-warning.mdx!)
 
 ## teleport
 
-The Teleport daemon is called `teleport`. It can be configured to run one or
-more "roles" with the `--roles` flags. The arguments to `--roles` correspond to
-the following services.
-
-<Notice type="tip" scope={["cloud"]}>
-
-Teleport Cloud manages Teleport instances with the `auth` and `proxy` roles. Use
-the remaining roles to manage access to specific resources and other Teleport
-clusters.
-
+The CLI tool that supports the Teleport access plane is called `teleport`, and allows Teleport services to be created, configured, and otherwise interacted with over the command line.
+The `teleport` process' CLI capability is generally the first step in setting up the Teleport binary following package manager or source installation on Linux.
+
+The primary commands for the teleport CLI are as follows:
+
+| Command | Description |
+| - | - |
+| `teleport help` | Outputs guidance for using commands on Teleport. |
+| `teleport start` | Starts the `teleport` service using the current shell session. This command will not start the `teleport` service as a daemon. |
+| `teleport status` | Prints the status of the current active Teleport SSH session. |
+| `teleport configure` | Generates and writes a [configuration YAML file](config.mdx) for the Teleport service. This file should be customized in production to suit the needs of your environment, and the default output should only be used when testing. |
+| `teleport version` | Prints the current release version of the Teleport binary installed on your system. |
+| `teleport app start` | Starts the application proxy service. |
+| `teleport db start` | Starts the database proxy service. |
+| `teleport db configure create` | Generates a database service configuration. |
+| `teleport db configure bootstrap` | Used to bootstrap a configuration to the Teleport database agent by reading a provided configuration. |
+| `teleport db configure aws print-iam` | Generates and outputs current IAM policies for a teleport managed database. |
+| `teleport db configure aws create-iam` | Generates, creates, and attaches desired IAM policies to a teleport managed database. |
+| `teleport install systemd` | Creates a systemd unit file, used to configure and install a `teleport` service daemon. |
+| `teleport node configure` | Generates a configuration file for a Teleport node accessed via SSH. |
```

Comment:
```suggestion
| `teleport node configure` | Generates a configuration file for a Teleport Node accessed via SSH. |
```
To distinguish this from the general term "node" (https://goteleport.com/docs/architecture/overview/#definitions).